### PR TITLE
docs: update FAQ. And modify release note.

### PR DESCRIPTION
**Pull request by HuiJu1218**
## Feat
- update  two FAQ.
- Modify release note page.

### CODE REVIEW

File: tocs/4.FAQ/FAQ.md [hunk 1]
```diff
@@ -102,3 +102,11 @@ Please load DPU accelerate application. Double check accelerate application list
 ```
 xmutil loadapp <dpu accelerate application name>
 ```
+
+# At first boot into system, what's the account name? password?
+Account name is `petalinux`. And you can set a new password at first time.
+
+# How to access USB?
+You can use command `sudo su` to switch into the `root mode` to access USB.
```
Comment (Shaing) [inline tocs/4.FAQ/FAQ.md]: You can use the command `sudo su` to switch mode to root for accessing USB.

File: tocs/4.FAQ/FAQ.md [hunk 2]
```diff
@@ -15,6 +15,8 @@
 - [How to update the application on X261?](#how-to-update-the-application-on-x261)
 - [How to check the verion on EXMU-X261?](#how-to-check-the-verion-on-exmu-x261)
 - [Aborted to run dpu-sc?](#aborted-to-run-dpu-sc)
+- [At first boot into system, what's the account name? password?](#at-first-boot-into-system-whats-the-account-name-password)
```
Comment (Shaing) [inline tocs/4.FAQ/FAQ.md]: In the first boot, what is the login account and password?

**Review by Shaing — CHANGES_REQUESTED** [review]
reviewed with comments.

File: tocs/4.FAQ/FAQ.md (hunk 2 above)
Comment (HuiJu1218) [reply tocs/4.FAQ/FAQ.md]: How about "For first time boot, what is the login account and password?"

File: tocs/4.FAQ/FAQ.md (hunk 1 above)
Comment (HuiJu1218) [reply tocs/4.FAQ/FAQ.md]: got it

**Review by Shaing — APPROVED** [review]

**Pull request merged**